addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6208,7 +6208,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#13435"
-msgid "Enable HQ Scalers for scalings above"
+msgid "Enable HQ scalers for scaling above"
```

Comment:
well, there is indeed only one "scaling" applied, but it applies for ALL scaling processes and different scaling factors, so one could argue it's still plural.
Maybe we should change the label to something like: "Use high quality upscaling for factors above"